kodev wbuilder fails to run

* KOReader version: Latest master (ee17b44b5)
* Device: Local, macOS 15.0.1 emulator

#### Issue

After following the developer guide, I am successfully able to build & run KOReader locally. However, running `./kodev wbuilder` results in the following error.
```
▸100% | Installing 'koreader'
[*] install front spec only for the emulator
[*] Install update once marker
[*] Install plugins
[*] Install resources
[*] Install data files
./?.lua;/usr/local/share/luajit-2.1/?.lua;/usr/local/share/lua/5.1/?.lua;/usr/local/share/lua/5.1/?/init.lua
./luajit: ./ffi/utf8proc.lua:15: attempt to call field 'loadlib' (a nil value)
stack traceback:
        ./ffi/utf8proc.lua:15: in main chunk
        [C]: in function 'require'
        frontend/util.lua:6: in main chunk
        [C]: in function 'require'
        frontend/luasettings.lua:8: in main chunk
        [C]: in function 'require'
        frontend/luadefaults.lua:6: in main chunk
        [C]: in function 'require'
        tools/wbuilder.lua:9: in main chunk
        [C]: at 0x01028e5410
make: *** [make/emulator.mk:26: run-wbuilder] Error 1
```

#### Steps to reproduce

1. Set up developer environment (for macOS) per documented instructions
2. Run `./kodev wbuilder`

##### `crash.log`

Verbose log:
```
[*] Install data files
! test -L koreader-emulator-arm64-apple-darwin24.0.0-debug/koreader/data || rm koreader-emulator-arm64-apple-darwin24.0.0-debug/koreader/data
install -d koreader-emulator-arm64-apple-darwin24.0.0-debug/koreader/data
gln -snfr base/thirdparty/kpvcrlib/crengine/cr3gui/data/NOTES.txt base/thirdparty/kpvcrlib/crengine/cr3gui/data/chm.css base/thirdparty/kpvcrlib/crengine/cr3gui/data/dict base/thirdparty/kpvcrlib/crengine/cr3gui/data/dict.css base/thirdparty/kpvcrlib/crengine/cr3gui/data/doc.css base/thirdparty/kpvcrlib/crengine/cr3gui/data/epub.css base/thirdparty/kpvcrlib/crengine/cr3gui/data/fb2.css base/thirdparty/kpvcrlib/crengine/cr3gui/data/htm.css base/thirdparty/kpvcrlib/crengine/cr3gui/data/html5.css base/thirdparty/kpvcrlib/crengine/cr3gui/data/hyph base/thirdparty/kpvcrlib/crengine/cr3gui/data/rtf.css base/thirdparty/kpvcrlib/crengine/cr3gui/data/tessdata base/thirdparty/kpvcrlib/crengine/cr3gui/data/txt.css base/build/arm64-apple-darwin24.0.0-debug/data/dict base/build/arm64-apple-darwin24.0.0-debug/data/tessdata base/thirdparty/kpvcrlib/cr3.css koreader-emulator-arm64-apple-darwin24.0.0-debug/koreader/data/
cd koreader-emulator-arm64-apple-darwin24.0.0-debug/koreader && EMULATE_READER_W=540 EMULATE_READER_H=720  ./luajit tools/wbuilder.lua
./?.lua;/usr/local/share/luajit-2.1/?.lua;/usr/local/share/lua/5.1/?.lua;/usr/local/share/lua/5.1/?/init.lua
./luajit: ./ffi/utf8proc.lua:15: attempt to call field 'loadlib' (a nil value)
stack traceback:
        ./ffi/utf8proc.lua:15: in main chunk
        [C]: in function 'require'
        frontend/util.lua:6: in main chunk
        [C]: in function 'require'
        frontend/luasettings.lua:8: in main chunk
        [C]: in function 'require'
        frontend/luadefaults.lua:6: in main chunk
        [C]: in function 'require'
        tools/wbuilder.lua:9: in main chunk
        [C]: at 0x0100bf1410
make: *** [make/emulator.mk:26: run-wbuilder] Error 1
```

--- a/tools/wbuilder.lua
+++ b/tools/wbuilder.lua
@@ -1,9 +1,7 @@
 -- widget test utility
 -- usage: ./luajit tools/wtest.lua
 
-print(package.path)
-package.path = "common/?.lua;frontend/?.lua;" .. package.path
-package.cpath = "common/?.so;common/?.dll;/usr/lib/lua/?.so;" .. package.cpath
+require("setupkoenv")
 
 -- Load default settings
 G_defaults = require("luadefaults"):open()

